pkgs/tools/security/pass/default.nix

```
@@ -1,34 +1,59 @@
-{ stdenv, lib, pkgs, fetchurl, buildEnv
-, coreutils, findutils, gnugrep, gnused, getopt, git, tree, gnupg, openssl
-, which, openssh, procps, qrencode, makeWrapper, pass
-
-, xclip ? null, xdotool ? null, dmenu ? null
-, x11Support ? !stdenv.isDarwin , dmenuSupport ? (x11Support || waylandSupport)
-, waylandSupport ? false, wl-clipboard ? null
-, ydotool ? null, dmenu-wayland ? null
-
-# For backwards-compatibility
-, tombPluginSupport ? false
+{
+  stdenv,
+  lib,
+  pkgs,
+  fetchurl,
+  buildEnv,
+  coreutils,
+  findutils,
+  gnugrep,
+  gnused,
+  getopt,
+  git,
+  tree,
+  gnupg,
+  openssl,
+  which,
+  openssh,
+  procps,
+  qrencode,
+  makeWrapper,
+  pass,
+
+  xclip ? null,
+  xdotool ? null,
+  dmenu ? null,
+  x11Support ? !stdenv.isDarwin,
+  dmenuSupport ? (x11Support || waylandSupport),
+  waylandSupport ? false,
+  wl-clipboard ? null,
+  ydotool ? null,
+  dmenu-wayland ? null,
```

```suggestion
  xclip,
  xdotool,
  dmenu,
  x11Support ? !stdenv.isDarwin,
  dmenuSupport ? (x11Support || waylandSupport),
  waylandSupport ? false,
  wl-clipboard,
  ydotool,
  dmenu-wayland,
```
Those are newer null unless you overwrite them on purpose and then you probably know what you are doing.

---

Remove the unneeded nulls. This is done on a separate commit just for the review: If approved, it will be squashed.